docs/src/Playground.tsx

```diff
@@ -198,13 +300,19 @@ function Playground() {
   React.useEffect(() => {
     if (play) {
       setEvents([])
-      void client.subscribe(new ${eventType}Subscription(['${symbolName}'])).then((subscription) =>
+      
+      const subscription = new ${eventType}Subscription(['${symbolName}']${
+                        stringifiedFieldsParam
+                            ? ", ".concat(stringifiedFieldsParam)
+                            : ""
+                    }${type === "timeSeries" ? `, ${dayStart}` : ""})
+      void client.subscribe(subscription).then((subscription) =>
         subscription.getStream().subscribe({
           onSubscribe: (sub) => {
             sub.request(Number.MAX_SAFE_INTEGER)
             unsubscribeRef.current = () => sub.cancel()
           },
-          onNext: (events) => events.forEach(handleEvent),
+          onNext: (events) => events.forEach(event => setTimeout(() => handleEvent(event), 0)),
```

Comment:
Why delayed?

Reply:
The amount of events may be tremendous, it just hangs the window. Event aggregated minute-wise Candles make handlers overflow event-loop.